### PR TITLE
Spanish language file partially updated

### PR DESCRIPTION
**Pull request by r01010010**
Didn't test anything

### CODE REVIEW

**Comment by jjeffryes** [comment]
Can you make these updates at https://www.transifex.com/ob1/openbazaar/ instead?

We pull the latest language files from there each time we do an update, that will overwrite any changes made directly in the repo. 

Thank you.

**Comment by r01010010** [comment]
Sure!

2016-11-03 16:36 GMT+01:00 Josh Jeffryes notifications@github.com:

File: js/languages/es.json [hunk 1]
```diff
@@ -62,10 +62,10 @@
     "Delete": "Borrar",
     "DontDelete": "No Borrar",
     "ConfirmDelete": "Confirmar eliminación",
-    "YesDelete": "Yes, delete",
+    "YesDelete": "Si, borrar",
     "Website": "Sitio Web",
     "Guid": "OpenBazaar ID",
-    "Welcome": "Bienvenido",
+    "Welcome": "Bienvenid@",
```
Comment (jjeffryes) [inline js/languages/es.json:68]: Should Bienvenid have an at sign at the end of it?

**Review by jjeffryes — CHANGES_REQUESTED** [review]
I noticed one translation that could have a typo in it.

**Comment by jjeffryes** [comment]
I'm closing this, as it will be overridden by the transifex file.

**Pull request closed without merging**